### PR TITLE
Updating React Native getting started docs for 0.60 and above

### PR DESCRIPTION
**Pull request by Ashish5591**
*Related Issues*
#949

*Description of changes:*
Updating React Native getting started docs for 0.60 and above

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: js/start.md [hunk 1]
```diff
@@ -223,9 +223,18 @@ $ react-native init myReactNativeApp
 $ cd myReactNativeApp
 $ npm install aws-amplify
 $ npm install aws-amplify-react-native
-$ react-native link
 ```
 
+If you are using React Native 0.60 and above, autolinking is enabled, but for iOS you would need to run the following step:
```
Comment (nikhil-dabhade) [inline js/start.md]: 1. Let's add a link for autolinking for readers who do not know what autolinking is : https://github.com/react-native-community/cli/blob/master/docs/autolinking.md

2. Not 100% clear on the instructions here: We are saying that for iOS you need to run the following steps even if they have React Native 0.60 ?

Comment (Ashish5591) [reply js/start.md]: 1 -> I agree. Added a link to the RN docs.
2 -> Yes thats right, on iOS you need to run the following steps if they have React Native 0.60.

Comment (nikhil-dabhade) [reply js/start.md]: For no. 2 let's phrase it -->
If you are using React Native 0.60 and above, autolinking is enabled. Run the following  commands for iOS:

Comment (Ashish5591) [reply js/start.md]: Updated!

**Pull request merged**